serving/docker/partition/properties_manager.py

```diff
@@ -134,6 +134,13 @@ def set_and_validate_entry_point(self):
                 if entry_point_file:
                     self.properties['entryPoint'] = 'model.py'
                 else:
-                    raise FileNotFoundError(
-                        f"model.py not found in model path {self.properties_dir}"
-                    )
+                    engine = self.properties['engine']
+                    if engine == "Deepspeed":
+                        entry_point = "djl_python.deepspeed"
+                    elif engine == "FasterTransformer":
+                        entry_point = "djl_python.fastertransformer"
+                    else:
```

Comment:
```
else if self.properties.contains("model_id"):
    entry_point = "djl_python.huggingface"
```

Reply:
How can we handle transformers-neuronx case?

Reply:
Can we do AOT in transformers-neuronx case?

Reply:
Not today, but sometime next week.
We need have a way to detect the script is running on inf2 docker image and on inf2 machine.

Reply:
Ah I see, Let me do that in the next PR. Also, no AOT for HF as of now.